browser/ntp_background/ntp_p3a_helper_impl.h

```diff
@@ -39,9 +39,9 @@ class NTPP3AHelperImpl : public NTPP3AHelper {
 
   void SetLastTabURL(const GURL& url) override;
 
-  void OnP3ARotation(brave::MetricLogType log_type);
+  void OnP3ARotation(brave::MetricLogType log_type, bool is_star);
 
-  void OnP3AMetricSent(const std::string& histogram_name);
+  void OnP3AMetricCycled(const std::string& histogram_name, bool is_star);
```

Comment:
I have no idea what `MetricCycled` means. Please add comments for these methods explaining what they do